### ISSUE TITLE
Ability to pass custom line length

### ISSUE THREAD

**Issue by IkBenOlie5** [parafoxia/len8#3]
Instead of the -l option extending the line length to 99 make it possible to give it a custom line length.
implementation:
```
$ len8 -l=130 my_project
$ len8 -l 130 my_project
```

**Comment by Jonxslays** [comment]
PEP 8 specifies a maximum of 99 characters which is why it is set up this way. ![image](https://user-images.githubusercontent.com/51417989/136687514-8c1377d8-0807-43ac-847f-c77ea2ac0a69.jpeg)

**Comment by Jonxslays** [comment]
it is a very easy change, i will let carbs decide on this one.

**Comment by IkBenOlie5** [comment]
If it'll be done maybe also allow for customizing comment line length

**Comment by parafoxia** [comment]
This main thing making me unsure about this is the project description:

> A utility for keeping line lengths within PEP 8 standards.

If we then allow people to go by things that aren't accepted in PEP 8, then that entire mission statement is basically void. It'd either be keep a hard line on it, or change up the overall purpose of len8 to just allow users to keep to whatever limits, and just have PEP 8 standards as the default.

**Comment by parafoxia** [comment]
We may revisit this at some point.

**Comment by parafoxia** [comment]
So after having `black`'s "manifesto" or sorts and finding out it defaults to 88 and not 79 like previously thought, we've decided to include a mechanism for supplying custom line lengths.

Planned implementation:
```sh
len8 .       # 79 chars
len8 . -l    # 88 chars
len8 . -ll   # 99 chars
len8 . -l x  # x chars
```

**Comment by parafoxia** [comment]
This is currently being implemented as part of #14. The `-l x` impl was not possible, however `-l` is additive, and `-c` and `-d` options have been added to set the lengths for code and docs respectively (note `-l` does not affect doc length).
